docs/apps/interchain-accounts/client.md

```diff
@@ -0,0 +1,232 @@
+<!--
+order: 7
+-->
+
+# Client
+
+## CLI
+
+A user can query and interact with the Interchain Accounts module using the CLI.
+
+### Controller
```

Comment:
I see your point, and I agree the CLI is pretty self documenting. But I have noticed from some users on Discord that sometimes it seems like they are not aware of the CLI commands we provide, so that's why I thought having something like this might help...

Reply:
Could we slim the docs down so if we make changes to the cli we don't have to update these docs as well (ie maintain docs in two places)?

I think it is fine to bring visibility to the CLI and explain how to obtain more information on how to use the CLI, but I'm sure an equal amount of individuals wouldn't realize there is more docs on the CLI outside of the self documentation obtainable via the `--help` command. I'd prefer to consolidate effort by making those commands have sufficient information

Reply:
All right, I trimmed down this file. I left explanation only for the `send-tx` and `generate-packet-data` command because I think there's an interplay between those 2 that is not immediately obvious from the command line. Please have a look!